--- pytorch/aten/src/ATen/cpu/vec/vec128/vec128_uint_aarch64.h	2026-01-27 18:00:34.052088780 +0800
+++ pytorch/aten/src/ATen/cpu/vec/vec128/vec128_uint_aarch64.h	2026-01-27 18:09:58.357501936 +0800
@@ -5,6 +5,7 @@
 #include <c10/macros/Macros.h>
 #include <c10/util/irange.h>
 
+typedef unsigned int uint;
 namespace at::vec {
 // Note [CPU_CAPABILITY namespace]
 // ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
